gitpod-io/website
# revamp license download page

## Related Issue(s)



Fixes #3195

## How to test

- Open [`/community-license/download`](https://deploy-preview-3203--gitpod-kumquat.netlify.app/community-license/download) & see 👀 



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3203"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
